jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -60,8 +65,7 @@ def process(self, input_table):
             # Save the output file
             if self.save_results:
                 self.save_model(
-                    output_model, suffix='wfscmb', output_file=outfile, format=False
-                )
+                    output_model, output_file=outfile, format=False)
```

Comment:
I think you do need to set the suffix somewhere, either right here when `save_model` is being called (as before), or include it as a param in the `spec` block above (https://github.com/spacetelescope/jwst/pull/6101/files#diff-07348f2cb4b6795e7d5f12d6d4741f5e33126ecc478ad35f67c20ccd5b56e041R17). Otherwise stpipe will likely use the default suffix based on the step name (wfscombine), which is not the official suffix expected for this product (downstream WFS software and the archive are expecting "wfscmb"). This step is setup differently than all other steps (which don't specific suffix names), because it's never run as part of a larger pipeline module. For steps that are run from a pipeline, the output data model is passed back to the pipeline and the pipeline takes care of saving it to a file. That's why you don't see this kind of operation in any other steps.

Reply:
When you run the step standalone, e.g. via the command line using "strun jwst.wfs_combine.WfsCombineStep <some_asn.json>" what kind of product type suffix does it use when writing the combined output image to disk? If `suffix` is not specified anywhere, I think it's going to use a generic file name suffix, rather than the "wfscmb" string that we want for this product. `suffix` should be added to the `spec` block up at line 22 and set to a default value of "wfscmb".

Reply:
I added the suffix of "wfscmb" to the block in line 23. I checked in the debugger and suffix is set correctly. The old value was "wfscombinestep". So, it was not horrible just not the expected value.